dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/tracking/AndroidXComponentPredicate.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.tracking
+
+import androidx.fragment.app.Fragment
+import com.datadog.android.rum.tracking.ComponentPredicate
+
+internal class AndroidXComponentPredicate(
```

Comment:
We shouldn't make this part of the default predicate. The number of app using Android TV is much smaller than those not targeting AndroidTv. As mentionned in the Jira ticket, we should create a dedicated `AndroidTvFragmentPredicate` in an `dd-sdk-android-tv` module instead.